app/src/main/java/com/otaman/weather/ui/forecast_report_screen/ForecastReportScreen.kt

```diff
@@ -29,9 +34,15 @@ private fun AppBar() {
             .background(color = Color.Transparent)
     ) {
         Button(
-            onClick = { /*TODO*/ },
             colors = ButtonDefaults.buttonColors(backgroundColor = Color.Transparent),
-            elevation = null
+            elevation = null,
+            onClick = {
+                navController.navigate(route = Screen.Main.route) {
+                    popUpTo(Screen.Main.route) {
+                        inclusive = true
+                    }
+                }
```

Comment:
Refactor: navigate back instead of navigating forward. Use [navigateUp](https://developer.android.com/reference/androidx/navigation/NavController#navigateUp()) or [popBackStack](https://developer.android.com/reference/androidx/navigation/NavController#popBackStack()) function